gadget/gadget_test.go

```diff
@@ -2793,7 +2803,7 @@ func (s *gadgetYamlTestSuite) TestLayoutCompatibilityTypeBareStructureAllowedMis
 
 	// ensure the first structure is barething in the YAML, but the first
 	// structure in the device layout is some-filesystem
-	c.Assert(gadgetLayout.LaidOutStructure[0].Type, Equals, "bare")
+	c.Assert(gadgetLayout.LaidOutStructure[0].GadgetStructure.Type, Equals, "bare")
```

Comment:
shouldn't this be the same now? I suppose this question applies to quite a few places?